Sources/Starknet/Data/Transaction/TransactionTrace.swift

```diff
@@ -104,13 +111,15 @@ public struct StarknetDeployAccountTransactionTrace: StarknetTransactionTrace {
     public let constructorInvocation: StarknetFunctionInvocation
     public let feeTransferInvocation: StarknetFunctionInvocation?
     public let stateDiff: StarknetStateDiff?
+    public let executionResources: StarknetExecutionResources
     public let type: StarknetTransactionType = .deployAccount
 
     private enum CodingKeys: String, CodingKey {
         case validateInvocation = "validate_invocation"
         case constructorInvocation = "constructor_invocation"
         case feeTransferInvocation = "fee_transfer_invocation"
         case stateDiff = "state_diff"
+        case executionResources = "execution_resources"
     }
 }
 
```

Comment:
There was a discussion on Slack that `execution_resources` should be added to `L1HandlerTransactionTrace` and it was also mentioned in https://github.com/software-mansion/starknet.py/pull/1312#discussion_r1525991404. Not sure if we want to add it as optional now or wait until it is fixed in the spec.

Reply:
Yeah, I thought it would be reasonable not to change this until the spec is updated, but I suppose making it optional won't hurt